packages/core/src/Modules/SmartTraderIFrame/SmartTraderIFrame.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const SmartTraderIFrame = () => (
+    <iframe
+        id='localstorage-sync'
+        src='https://aaron-binary3.binary.sx/br_ls-sync/localstorage-sync.html'
```

Comment:
are we committing your fork?

Reply:
can we commit what we anticipate the domain being?

Reply:
with a TODO to check it and update it in the future

Reply:
Updated. 👍